kdesk/icon-hooks.sh

```diff
@@ -102,30 +102,29 @@ case $icon_name in
 
     "world")
         IFS=$'\n'
-        kano_statuses=`kano-profile-cli get_notifications_count`
-        is_online=`kano-profile-cli is_registered`
+        # Returns if online, prints notification count and stats activity
+        info=`kano-profile-cli get_world_info`
         apirc=$?
 
         if [ "$debug" == "true" ]; then
-            printf "Kano Profile API returns rc=$apirc, data=\n$kano_statuses\n"
+            printf "Kano Profile API returns rc=$apirc, data=\n$info\n"
```

Comment:
@skarbat indeed performance would be hindered by this. Ignore that comment then :-)